src/config/settings/base.py

```diff
@@ -322,17 +322,19 @@
 # Add additional configuration below:
 # ------------------------------------------------------------------------------
 
-# Django-machina search backend:
+# DJANGO-MACHINA SETTINGS
+# ------------------------------------------------------------------------------
 # https://django-machina.readthedocs.io/en/latest/getting_started.html#django-haystack-settings
 HAYSTACK_CONNECTIONS = {
     "default": {
-        "ENGINE": "haystack.backends.simple_backend.SimpleEngine",
+        "ENGINE": "haystack.backends.whoosh_backend.WhooshEngine",
+        "PATH": str(BASE_DIR / "chigame/forums/search_index"),
     },
 }
 
+# https://django-haystack.readthedocs.io/en/latest/signal_processors.html
```

Comment:
I would add a comment briefly mentioning what a "signal processor" is and why you chose this one.